frame/identity/src/lib.rs

```diff
@@ -291,6 +291,7 @@ pub enum IdentityField {
 	PgpFingerprint = 0b0000000000000000000000000000000000000000000000000000000000100000,
 	Image          = 0b0000000000000000000000000000000000000000000000000000000001000000,
 	Twitter        = 0b0000000000000000000000000000000000000000000000000000000010000000,
+	Totem          = 0b0000000000000000000000000000000000000000000000000000000100000000,
```

Comment:
Why did you added these changes?

Reply:
@bkchr The Totem Live Accounting Public UI has a p2p messaging interface that establishes a user identity which is seperate from the Substrate address(es) that are used sign & make transactions. 

It allows us to route p2p messaging for example for sharing off-chain information between users. Eventually these messages will be e2e encrypted.

The userid is created during the on-boarding process (Getting started Module for Totem (https://totem.live) in addition to the initial Substrate address generation using polkadotjs).

This messaging userid is intended to be shared in public without revealing associated addresses under the control of that user it has not direct connection to the substrate keys at all. 

As it can be shared in public In that respect it could be used in the context of validators identifying themselves on the Polkadot network for example as is done with Twitter and email config in this module.

There is one important distinction for our userid: we also are implementing an on-chain key registry which stores public message signing keys _and_ public encryption keys associated with the messaging userid. This is partially implemented in our chain but not yet connected to the UI.

The use case we see which justifies this change is that in principle this will allow any other party to validate that a user claiming to have a Totem handle (and who has a signed a proof/message from them), is who they say they are by reading the key registry from Totem with the supplied userid as key to the storage and doing a comparison.

This will allow them to establish both that the userid exist and that it was signed with the same signing key associated with that userid. Twitter and email cannot do this.

We see this as a sort of combination of using of a public twitter handle and a public pgp key but done in a very _"substratey"_ way. 

As we are building with Substrate since 2018, our view was that this change should be added to the network identity pallet but we did not consider this as a breaking change. So we apologise for this.

We are also aware that should this change is accepted it will also imply a chainge to `polkdotjs/api` to align.

Happy to discuss/answer any more questions you might have on this.

Reply:
I think you should bring this up in a different pr.

Reply:
@chrisdcosta If you want to use our identity pallet for this purpose, you should use the "addtional" data field to store any arbitrary information you want:

```
	/// Additional fields of the identity that are not catered for with the struct's explicit
	/// fields.
	pub additional: Vec<(Data, Data)>,
```

A change like this would introduce the `totem` identifier to every substrate chain that uses our out of the box identity pallet, and this is not very general, and probably not something most teams would want.

If you want to support a feature like this more directly, you can integrate your own identity pallet with this kind of change, but probably not right to push this upstream.

Reply:
@bkchr oh, that commit should not have been included yet. That's a mistake. Sorry I responded to your screenshot, and didn't refer to the logs here. I will review and try to resolve.